app/styles/_data-toolbar.less

```diff
@@ -1,6 +1,13 @@
 /* Data toolbar
    - Includes customizations for label filter input and active filter
 ---------------------------------------------------------------------------- */
+// @screen-md-min - (sidebar-left + left + right margin); 992 - (145 + 30 + 30) = 787
+@data-toolbar-filter-max-width:   (@screen-md-min - (@sidebar-left-width-md + @middle-content-container-padding-lg + @middle-content-container-padding-lg));
+@label-filter-default-input-lg: (@label-filter-default-input-md + 190); // 730px
+@label-filter-default-input-md: (@label-filter-default-input-sm + 140); // 540px
+@label-filter-default-input-sm: (@label-filter-default-input-xs + 160); // 400px
```

Comment:
If you keep the variables names, maybe use `label-filter-input-width-*`? `-width` should be in there somewhere.